revolv/static/dashboard.scss

```diff
@@ -532,10 +532,108 @@ $sidebar-icon-gray: #999;
 
 $light-gray: #c6c6c6;
 $dark-gray: #555555;
+$icon-height: 100px;
 
 .dashboard-data-section-my-impact {
     height: 100%;
 
+    .my-impact-container {
+
+        .statistics-container {
+
+            .icon-container {
+                padding: 0;
+            }
+
+            .contribution-text {
+                font-weight: 800;
+                margin: 0 10% 40px 10%;
+                text-align: center;
+                font-size: 1.05rem;
+                margin-top: 50px;
+                color: $dark-gray;
+            }
+
+            .icon {
+                height: $icon-height;
+                margin: 0;
+            }
+
+            .statistics-row {
+
+                .statistic {
+                    margin-bottom: 20px;
+                }
+
+                .description  {
+                    font-weight: 300;
+                    line-height: 1rem;
+                    margin: 0;
+                }
+
+                .emphasized-text {
+                    font-weight: 800;
+                    font-size: 1.3rem;
+                    //line-height: 1.15rem;
+                    margin: 0;
+
+                    &.repayments {
+                        color: $revolv-blue;
+                    }
+
+                    &.number-of-projects {
+                        color: #fdb131;
```

Comment:
I think for colors it's good to add them to settings, just so that if someone asks for all the colors we're using from a design standpoint, we can give them to them in one place (this happened with RE-volv a few weeks ago). Also, the same colors will be used in #192 so we'll have to do it eventually.